var/spack/repos/builtin/packages/suntans/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Suntans(MakefilePackage):
+    """The Stanford unstructured-grid, 
+    nonhydrostatic, parallel coastal 
+    ocean model. """
+
+    homepage = "https://github.com/ofringer/suntans"
+    url      = "https://github.com/ofringer/suntans/archive/master.zip"
+
+    maintainers = ['ofringer', 'zyaj', 'mrayson', 'jadelson', 'lhxone']
+
+    version('0.0.1', '9b5d0dee5502a5650e37bf72f64b1124be86c084e7408aa517c214ddb25e36de')
```

Comment:
A question and a comment:
1. Why the version is `0.0.1` when the tags in the repository say `v3.1` was tagged in 2015?
2. If we retrieve from a git repository it would be better to pin the commit sha

Reply:
Sorry for that, `0.0.1` is a test version name, and I have already fixed the above.